### ISSUE TITLE
Failure to load file

### ISSUE THREAD

**Issue by SimonSolar2C** [supermerill/SuperSlicer#2482]
### Version
_Version of SuperSlicer used goes here (help->about)_
Version: 2.3.56.8
Build: SuperSlicer_2.3.56.8
### Operating system type + version
_What OS are you using, and state any version #s_
Windows10 64bit

### Behavior
* _Describe the problem_ConfigBase::set_deserialize() failed for parameter "fill_pattern", value ""
* _Steps needed to reproduce the problem_Open the 3mf project file results in error message. I dont know what version saved this file
* _Expected Results_Project opens and loads
* _Actual Results_Error message
  * _Screenshots if useful_
![image](https://user-images.githubusercontent.com/12725844/156882332-e25e50ba-5c15-4333-8a25-e09066be569b.png)

_Is this a new feature request?_No
Related guides for writing feature requests: http://meta.stackexchange.com/a/259196 http://nickohrn.com/2013/09/write-great-feature-request-bug-report/

#### Project File (.3MF) where problem occurs
_*Upload a Project File* (.3MF) `Plater -> Save Project as...`_
_You have to zip it (or rename it to .zip) to be able to drag& drop it here
[V5e2Finalx2BW.zip](https://github.com/supermerill/SuperSlicer/files/8190631/V5e2Finalx2BW.zip)
_

**Comment by mojamuto** [comment]
It seems to be the same problem: #2481

**Comment by SimonSolar2C** [comment]
Yes probably the same problem (slightly different message) although it seems it was closed without a solution being posted.

**Comment by supermerill** [comment]
probably duplicate of #1983
